src/txHashBuilder.c

```diff
@@ -1434,6 +1442,8 @@ static void txHashBuilder_assertCanLeaveValidityIntervalStart(tx_hash_builder_t*
 
 // ============================== MINT ==============================
 
+#ifndef APP_XS_DISABLE_TOKEN_MINTING
+
```

Comment:
That's because there is no obligation to define the function. Declaring a prototype is OK. Only the linker detects that a function def is missing (could be in another .c file, so compiler does not care), and it only complains if that function is actually called AFAIK.